lib/aws.py

```diff
@@ -256,6 +257,29 @@ def instance_outdated_launchtemplate(instance_obj, asg_lt_name, asg_lt_version):
     return False
 
 
+def older_instance (instance_id, days_fresh):
+
+    response = ec2_client.describe_instances(
+        InstanceIds=[
+            instance_id,
+        ]
+    )
+
+    instance_launch_time = response['Reservations'][0]['Instances'][0]['LaunchTime']
+
+    instance_age = ((datetime.datetime.now(instance_launch_time.tzinfo) - instance_launch_time).days)
+    instance_age_remainder = ((datetime.datetime.now(instance_launch_time.tzinfo) - instance_launch_time).seconds)
+
+    if instance_age > days_fresh:
+        logger.info("Instance id {} launch age of '{}' day(s) is older than expected '{}' day(s)".format(instance_id, instance_age, days_fresh))
+        return True
+    elif (instance_age == days_fresh) and (instance_age_remainder > 0):
```

Comment:
That logic is not very clear to read. 
I understand that you check the number of days and if it's the same you check the seconds. Why don't you just check the seconds to begin with? And just show the offset based on the result.

Either way I feel the code could be made a bit clearer. Maybe with a comment or by renaming these variable to carry what you're doing.

Reply:
Honestly @kuuji either I don't know how to use this library or it's not the appropriate library for this `datetime`, but somehow a difference in time is returning `xx days xxxxx seconds`.

Initially I was testing using only `.seconds` of the difference and it was missing nodes days older than limit, then I realized that was returning the `xxxxx seconds` of the time diff. 

So that's why I have something like `instance_age` which is the diff in days and when that's equal the  `days_fresh` EC2 age limit I look for the `instance_age_remainder` in seconds.

Do you feel comments here might make it clearer?

Reply:
I put comments on `instance_age` and `instance_age_remainder`, let me know if that clarifies the purpose of this?